services/ssp-b/src/index.js

```diff
@@ -0,0 +1,311 @@
+//       const debug_key = debugKey()
+//       const AttributionReportingRegisterSource = {
+//         destination,
+//         source_event_id,
+//         debug_key,
+//         aggregation_keys: {
+//           quantity: sourceKeyPiece({
+//             type: SOURCE_TYPE["view"], // view attribution
+//             advertiser: ADVERTISER[advertiser],
+//             publisher: PUBLISHER["news"],
+//             id: Number(`0x${id}`),
+//             dimension: DIMENSION["quantity"]
+//           }),
+//           gross: sourceKeyPiece({
+//             type: SOURCE_TYPE["view"], // view attribution
+//             advertiser: ADVERTISER[advertiser],
+//             publisher: PUBLISHER["news"],
+//             id: Number(`0x${id}`),
+//             dimension: DIMENSION["gross"]
+//           })
+//         }
+//       }
+
+//       console.log("Registering event source :", { AttributionReportingRegisterSource })
+//       res.setHeader("Attribution-Reporting-Register-Source", JSON.stringify(AttributionReportingRegisterSource))
+//       res.status(200).send("attribution event (view) source registered")
+//     } else {
+//       res.status(400).send("'Attribution-Reporting-Eligible' header is malformed") // just send back response header. no content.
+//     }
+//   } else {
+//     res.status(400).send("'Attribution-Reporting-Eligible' header is missing") // just send back response header. no content.
+//   }
+// })
+
+// app.get("/register-trigger", async (req, res) => {
+//   const { id, quantity, size, category, gross } = req.query
+
+//   const AttributionReportingRegisterTrigger = {
+//     aggregatable_trigger_data: [
+//       {
+//         key_piece: triggerKeyPiece({
+//           type: TRIGGER_TYPE["quantity"],
+//           id: parseInt(id, 16),
+//           size: Number(size),
+//           category: Number(category),
+//           option: 0
+//         }),
+//         source_keys: ["quantity"]
+//       },
+//       {
+//         key_piece: triggerKeyPiece({
+//           type: TRIGGER_TYPE["gross"],
+//           id: parseInt(id, 16),
+//           size: Number(size),
+//           category: Number(category),
+//           option: 0
+//         }),
+//         source_keys: ["gross"]
+//       }
+//     ],
+//     aggregatable_values: {
+//       // TODO: scaling
+//       quantity: Number(quantity),
+//       gross: Number(gross)
+//     },
+//     debug_key: debugKey()
+//   }
+//   res.setHeader("Attribution-Reporting-Register-Trigger", JSON.stringify(AttributionReportingRegisterTrigger))
+//   res.sendStatus(200)
+// })
+
+app.get('/ad-tag.html', async (req, res) => {
+  res.render('ad-tag.html.ejs');
+});
+
+app.get('/video-ad-tag.html', async (req, res) => {
+  res.render('video-ad-tag.html.ejs');
+});
+
+// app.get("/reports", async (req, res) => {
+//   res.render("reports.html.ejs", { title: "Report", Reports })
+// })
+
+app.get('/auction-config.json', async (req, res) => {
```

Comment:
you'll probably don't need this if you are using SSP-A as a component seller only (Component auction config is provided by top-seller right ?)

else you may want to update the const

Reply:
this is updated in the next PR. i'll leave this one alone for now bc if i remove it, it will make the rebase really annoying.